code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -993,17 +993,17 @@
 	description = "Causes heavy toxin damage after a brief time of inactivity."
 	reagent_state = LIQUID
 	metabolization_rate = 0 //stays in the system until active.
-	var/actual_metaboliztion_rate = REAGENTS_METABOLISM
+	var/actual_metaboliztion_rate = 0.5 * REAGENTS_METABOLISM
```

Comment:
?

Reply:
This was done before I messed with REAGENTS_METABOLISM